torch/ao/quantization/_dbr/mappings.py

```diff
@@ -23,7 +23,12 @@
     torch.mul: torch.ops.quantized.mul,
     operator.mul: torch.ops.quantized.mul,
     torch.cat: torch.ops.quantized.cat,
+    F.conv1d: torch.ops.quantized.conv1d,
     F.conv2d: torch.ops.quantized.conv2d,
+    F.conv3d: torch.ops.quantized.conv3d,
+    F.conv_transpose1d: torch.ops.quantized.conv_transpose1d,
```

Comment:
I see that the tests cover `F.conv1d` and `F.conv3d`, are the transpose variants tested as well?

Reply:
Yes, I'm adding the tests now. They didn't work out of the box and I need to look into why.